contacts_kickstart.info.yml

```diff
@@ -9,6 +9,7 @@ distribution:
 dependencies:
   # Core
   - toolbar
+  - system
```

Comment:
Nope, all of the contacts module requirements assume that system is enabled by the profile...